### ISSUE TITLE
How to attach plugin to Wrapper Vue instance?

### ISSUE THREAD

**Issue by ale-grosselle** [vuejs/vue-web-component-wrapper#39]
Hi, 
I am trying to use this library for my Vue components. All of my exportable components use "vue-i18n" adding i18n property to item:

```
...
i18n: {
		messages: Langs
	}
```

In web-component-wrapper, when instance Vue object:
```
const wrapper = this._wrapper = new Vue({....})
```
I can't understand how to pass to vue instance the vueI18n instance. 

Only for test, if I add: 
```
	    i18n: {locale: 'IT'}
```
as property to Vue instance, all works as expected.

**Comment by timAtStrathcom** [comment]
I came here to ask this exact question. I am building a webcomponent using vue-cli 3 `--target wc` and I need to use vue-i18n. Is there a way to do this?

**Comment by karol-f** [comment]
Didn't test it but can't you just add `i18n` property to component object? Same as you would do when initiating whole app - but on component level.

**Comment by timAtStrathcom** [comment]
@karol-f yes, that sort of works. It gets the plugin initialized for the component but, at least in the case of i18n, there seems to be some initialization not happening because none of my translation strings work like that. Maybe that becomes a problem with vue-i18n at that point.

**Comment by karol-f** [comment]
@timAtStrathcom It's working for me - https://codesandbox.io/s/jzwm57k0wv . It's not Custom Element but You get the idea - please check App.vue.

**Comment by ale-grosselle** [comment]
I think your solution does not work with a component that it has some
children  components.
Nested component can't get the translation.

Il giorno lun 19 nov 2018, 20:46 Karol Fabjańczuk <notifications@github.com>
ha scritto:

**Comment by timAtStrathcom** [comment]
Yes, that's exactly the problem. I've added another component to the example. It displays strings as the default `en-US`: https://codesandbox.io/s/xo1911q2nq

**Comment by ale-grosselle** [comment]
Yes, but with your solution I must to define all translation from the
parent.
All my component define the translation string:
https://kazupon.github.io/vue-i18n/guide/component.html#translation-in-functional-component

I think It could Be a good idea if we can pass to the define method the
plugins list to attach to vue instance

Il giorno lun 19 nov 2018, 21:22 Karol Fabjańczuk <notifications@github.com>
ha scritto:

**Comment by timAtStrathcom** [comment]
@karol-f Thanks so much for spending time helping with this! I can find lots of ways to completely break vue-i18n in a web component but, as you've shown, as long as every component defers to the entry component then things seem to work acceptably.

As @ale-grosselle mentions, it still might be nice if there was a way to affect the creation of the root Vue instance. This would help with support of vue-i18n's more advanced features, like the SFC <i18n> tag, local component overrides, etc. Not sure how feasible it would be do that or how many other plugins might benefit.

**Comment by EranGrin** [comment]
I was also struggling with this issue for quite some time and left it with a workaround, but finally, I took time to dive into this and refactor that workaround.
I added my solution over here https://stackoverflow.com/questions/53341395/how-can-i-use-vue-i18n-in-a-vue-web-component/68734331#68734331
